src/main/java/frc/robot/subsystems/swerve/modules/ModuleConstants.java

```diff
@@ -6,14 +6,22 @@
 import frc.utils.Conversions;
 import frc.utils.battery.BatteryUtils;
 
-public record ModuleConstants(double wheelDiameterMeters, double couplingRatio, Rotation2d velocityAt12VoltsPerSecond) {
+public record ModuleConstants(String swerveLogPath, String modulesName, double wheelDiameterMeters, double couplingRatio, Rotation2d velocityAt12VoltsPerSecond) {
 
-    public ModuleConstants(double wheelDiameterMeters, double couplingRatio, double velocityAt12VoltsMetersPerSecond){
-        this(wheelDiameterMeters, couplingRatio, Conversions.distanceToAngle(velocityAt12VoltsMetersPerSecond, wheelDiameterMeters));
+    public ModuleConstants(String swerveLogPath, String modulesName, double wheelDiameterMeters, double couplingRatio,
+            double velocityAt12VoltsMetersPerSecond){
+        this(swerveLogPath, modulesName, wheelDiameterMeters, couplingRatio,
+                Conversions.distanceToAngle(velocityAt12VoltsMetersPerSecond,
+                wheelDiameterMeters));
     }
 
-    static final String LOG_PATH = SwerveConstants.SWERVE_LOG_PATH + "Modules/";
-    static final String ALERT_LOG_PATH = LogPaths.ALERT_LOG_PATH + LOG_PATH;
+    String getLogPath() {
+        return swerveLogPath() + modulesName() + "Modules/";
+    }
+
+    String getAlertLogPath() {
+        return LogPaths.ALERT_LOG_PATH + getLogPath();
+    }
```

Comment:
use record not get, also no need to save alert log path (if you thought about doing it)